text/0000-translateable-ui.md

```diff
@@ -0,0 +1,128 @@
+# Table of Contents
+
+- [Tracking Issues](#tracking-issue)
+  - [Amethyst Community Forum Discussion](#forum-discussion)
+- [Motivation]
+- [Guide Level Explanation](#guide-level-explanation)
+- [Reference Level Explanation](#reference-level-explanation)
+- [Drawbacks]
+- [Rationale and Alternatives](#rationale-and-alternatives)
+- [Prior Art](#prior-art)
+- [Unresolved Questions](#unresolved-questions)
+
+# Basic Info
+[basic]: #basic-info
+
+- Feature Name: translateable_prefabs
+- Start Date: 2018-11-19
+- RFC PR: 
+- [Tracking Issue](#tracking-issue): 
+- [Forum Thread](#forum-discussion): No.
+
+After filling out these fields, please proceed to fill out the remaining sections of this document.
+
+# Summary
+[summary]: #summary
+
+Currently, Amethyst is able to provide prefabs for component hierarchies, for example UIs.
+They should predefine what the engine loads and displays in a convenient manner.
+Amethyst can also load locales using Fluid, which allows the translation of an application into many languages with ease.
```

Comment:
Isnt it fluent and not fluid?

Reply:
ah, I completely missed that one, you are right!